importer/application/src/commands/tmdb/subcommands/import.ts

```
@@ -0,0 +1,92 @@
+        }
+
+        const toInsert = toInsertRaw.map((obj) => obj.data!);
+
+        const res = await collection.insertMany(toInsert).catch((err) => {
+          return logger.error('Failed to insert movies', { err });
+        });
+
+        if (res) {
+          state = {
+            ...state,
+            ...toInsertRaw.reduce((acc, obj) => ({ ...acc, ...obj.state }), {}),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)